### ISSUE TITLE
[BUG] MapsSearchClient.GetReverseGeocodingAsync(): CountryRegion.Iso is null.

### ISSUE THREAD

**Issue by julien-bertolini** [Azure/azure-sdk-for-net#46313]
### Library name and version

Azure.Maps.Search 2.0.0-beta3

### Describe the bug

MapsSearchClient.GetReverseGeocodingAsync() always returns null ISO country code:
![image](https://github.com/user-attachments/assets/f07b365b-4c24-4c7c-821f-9cc9ca2d58f1)

In the RawResponse content, we can see that ISO is correct. :
![image](https://github.com/user-attachments/assets/3cc64100-e503-402e-b607-75fa4cac045a)

### Expected behavior

ISO Country Code should be the same as the one in the raw response content.

### Actual behavior

ISO Country Code is always null.

### Reproduction Steps

```
var geopos = new GeoPosition(0.377955, 42.807553);
var response = await _searchClient.GetReverseGeocodingAsync(geopos);
var address = response.Value.Features.FirstOrDefault()?.Properties.Address;
```

**Comment by dubiety** [comment]
Hi @julien-bertolini ,

Thank you for reporting this issue. We're fixing this issue and will get back to you once this fix is released!

**Comment by dubiety** [comment]
Hi @julien-bertolini , It's fixed in our 2.0.0-beta.4 . I'll close this bug for now. Let us know if there's any problem. Thank you very much!
